script.js

```diff
@@ -0,0 +1,188 @@
+"use strict";
+
+// import { Modal } from "modal.js"
+
+window.onload = function () {
+  //Add Tags
+  addTagsClickHandler();
+
+  //showing and hiding of mobile screens
+  showAndHidePhoneDisplay();
+
+  //screen slider
+  slideScreensWithPhones();
+};
+
+const addTagsClickHandler = () => {
+  document.querySelector(".portfolio__tags").addEventListener("click", (e) => {
```

Comment:
if you add eventListener you need to remove it somewhere

Reply:
const addTagsClickHandler = () => {
  document.querySelector(".portfolio__tags").addEventListener("click", (e) => {
    if (e.target.getAttribute("class") === 'tag') {
      let clickedTag = e.target;
      removeSelectedTags();
      selectClickdeTag(clickedTag);
      changeImageOrder()
    }
    document.querySelector('.portfolio__tags').removeEventListener('click', (e))
  });
};


?????

Reply:
no, because your handler will not work any more. to be honest in this case you can leave without removing but in spa  you should be very careful with adding and removing events if you remove elements without reloading the page
https://developer.mozilla.org/ru/docs/Web/Events